assets/src/editor/events/tickets/editor-ticket/filter-bar/with-ticket-entities-for-filtered-date-entities.js

```diff
@@ -11,10 +10,18 @@ const withTicketEntitiesForFilteredDateEntities = createHigherOrderComponent(
 		{
 			filteredDateEntities = DEFAULT_EMPTY_ARRAY,
 			ticketEntities = DEFAULT_EMPTY_ARRAY,
+			isChained = true,
 		}
 	) => {
-		const dateIds = getDateEntityIds( filteredDateEntities );
+		if ( ! isChained ) {
+			return { ticketEntities };
+		}
```

Comment:
So the nice thing here, is if the ticket entity list is not chained to the date entity list, then any date state updates are not going to have any effect on the ticket entity list.

Reply:
what about date ticket relations though?

for example, if the ticket list is unchained and I add or remove ticket assignments for a particular datetime. Would the "item count" (the little number bubble that appears over the calendar icon in the ticket action menu) for the affected tickets change?

Reply:
> for example, if the ticket list is unchained and I add or remove ticket assignments for a particular datetime. Would the "item count" (the little number bubble that appears over the calendar icon in the ticket action menu) for the affected tickets change?

Yes it will because that bubble is reacting now to _state_ change in the store, not passed through props.

Reply:
Oh wait, I think I might have misunderstood.  Is the expectation that if `isChained` that the bubble only on the ticket only shows the count of dates matching the dates in the _filtered date list_ (even if the ticket may be assigned to more dates)?  If so, I'm not sure it currently behaves that way.  However, if that's the way yo _do_ want it, then it should be fairly trivial to simply ensure we use `withFilteredDatetimes` in the composite for the component displaying that count as a part of the Ticket Assignments Manager Modal.  I'd like to implement/fix that in a separate pull if needed?

Reply:
> Is the expectation that if isChained that the bubble only on the ticket only shows the count of dates matching the dates in the filtered date list (even if the ticket may be assigned to more dates)?

oh no, that's not what I meant at all.

I think if the ticket is assigned to 10 dates, then the item count bubble should read 10 regardless of whether any of those related dates are displayed in the dates list or not.

My question was whether that item count bubble on a ticket would decrement to 9 if you removed that ticket assignment via the date when the ticket list was unchained.

Reply:
> My question was whether that item count bubble on a ticket would decrement to 9 if you removed that ticket assignment via the date when the ticket list was unchained.

Ahh ok, then in that case yes it will update correctly because the bubble count is not relying on passed through props but getting the number of dates related to the ticket via the store state.

Reply:
perfect